### DISCUSSION TITLE
How about the same "double" comparison like in Python?

### DISCUSSION THREAD

**Discussion by olivierpons** [godotengine/godot-proposals#5078] [Scripting]
Would it be possible to write:

```
if PI < $mesh.rotation.z < 0:
    pass

if 0 > $mesh.rotation.x > 0:
    pass
```

Still the same (but easier to read) than this:

```
if PI < $mesh.rotation.z and $mesh.rotation.z < 0:
    pass

if 0 > $mesh.rotation.x and $mesh.rotation.x > 0:
    pass
```

**Comment by mieldepoche** [comment]
There is already a proposal for this: https://github.com/godotengine/godot-proposals/issues/889

**Comment by Mickeon** [comment]
And also a discussion! https://github.com/godotengine/godot-proposals/discussions/4004
